rules/concepts.md

```diff
@@ -22,4 +23,59 @@ formerly implicit variables
 How to find and install.
 How to write will be on a later page.
 
-### Comprehensive Examples
\ No newline at end of file
+### Comprehensive Examples
+-->
+
+# Rules Concepts
+
+Thus far we've connected openHAB to devices through Things, modeled the devices with Items, discussed persistence and how to build the display to control your home.
+But all of that amounts to home control, not automation.
+To create home automation we need to define rules.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+## What Are Rules
+
+You can think of rules as routines or behaviours for your smart home.
+
+Just like you stand up in the morning and have your breakfast, your smart home can have a routine to open the blinds when the sun rises.
+Another routine would be to turn all lights off when you leave home: openHAB can do that for you, no need to open your app and switch of the lights manually.
+You want to hear your favorite music when you arrive back at home? No problem, openHAB can do that for you.
+But rules can also remind you of things: you opened the window hours ago and forgot that is is open? openHAB can sent a notification to your phone.
+
+To summarize in an universal principle: *When __a__ happens, than do __b__*.
+For many cases, it can be practical to extend that to *When __a__ happens, than do __b__ but only if __c__*.
+
+To take some examples from above, you can have openHAB to only turn on your favorite music when you arrive at home if it is not your kids bedtime.
+You do not want openHAB to open your blinds on the weekend because you want to sleep a little bit longer? No problem, *but only if* helps.
+
+## Parts of a Rule
+
+To work with the *When __a__ happens, than do __b__ (but only if __c__)* principle, openHAB rules consist of three parts:
+
```

Comment:
Presumably you mean the parens to indicate it's optional. If so, the when part also needs to be in parens. 

Somewhere we can add something like 

> Why would one want to have a rule without triggers? In some cases it is useful to have a rule that is manually triggered. In other cases it's useful to have one rule call another rule. In both of these cases, there is no event that triggers the rule.